packages/@webex/plugin-meetings/src/interpretation/index.ts

```diff
@@ -177,6 +194,125 @@ const SimultaneousInterpretation = WebexPlugin.extend({
       throw error;
     });
   },
+  /**
+   * Sets up a listener for handoff requests from mercury
+   * @returns {void}
+   */
+  listenToHandoffRequests() {
+    this.listenTo(this.webex.internal.mercury, 'event:locus.approval_request', (event) => {
+      if (event?.data?.approval?.resourceType === INTERPRETATION.RESOURCE_TYPE) {
+        const {receivers, initiator, actionType, url} = event.data.approval;
+        const receiverId = receivers?.[0]?.participantId;
```

Comment:
why do we get the first element here, but the next link we check if there are other receivers and check if its yourself  ?

Reply:
Oh, yes, you're right, in this event, should only one receiver, so I just use` receivers[0]` to get the receiver id, I will change the check whether is self logic to just use the `receiverId`